pages/api/candidateApiClient.module.ts

```diff
@@ -36,3 +41,16 @@ export async function getCandidateTests() {
         return error.message;
     }
 }
+
+export async function addTestSubmisson( tokenId: string, newTestSubmission: NewTestSubmission) {
+    const {publicRuntimeConfig} = getConfig();
+    const apiURL = publicRuntimeConfig.API_URL;
+
+    return await fetch(`${apiURL}/sessions/${tokenId}`, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify(newTestSubmission),
+    });
+}
```

Comment:
I _think_ this is part of a different ticket that Hema has been working on.
Have you talked to her about it?

Reply:
also it looks like we don't use this anywhere?

Reply:
Yes, just thinking out loud. Will remove...